br/pkg/task/operator/force_flush.go

```diff
@@ -0,0 +1,99 @@
+package operator
+
+import (
+	"context"
+	"crypto/tls"
+	"slices"
+
+	"github.com/pingcap/errors"
+	logbackup "github.com/pingcap/kvproto/pkg/logbackuppb"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tidb/br/pkg/task"
+	"github.com/pingcap/tidb/br/pkg/utils"
+	"github.com/pingcap/tidb/pkg/util/engine"
+	pd "github.com/tikv/pd/client"
+	"github.com/tikv/pd/client/opt"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+)
+
+func getAllTiKVs(ctx context.Context, p pd.Client) ([]*metapb.Store, error) {
```

Comment:
i think it's not a good idea for production system to force flush to all tikv nodes. is it possible to expose the interface to do individual tikv level flush?

Reply:
There is, you may use the `--stores <regexp>` to select stores to be flushed by their addresses. (Default is all TiKV).